documentation/howto/os3_remote.adoc

```diff
@@ -1,107 +1,96 @@
-= Set up and Remotely Monitor an OpenShift Database Server
+= Set up and Remotely Monitor an OpenShift Application
 :page-layout: howto
 :page-tab: docs
 :page-status: green
 :experimental:
 :imagesdir: ./images
-:toc:
 
-In some scenarios, the user may already have a remote instance of OpenShift 3 running with various applications on it and wants to monitor it. Red Hat JBoss Developer Studio allows users to set up a connection to a remote instance of OpenShift 3 and then use logs (such as the application and build logs) to troubleshoot and monitor running applications. Use the following tasks to connect to and work with a remote OpenShift instance:
+In some scenarios, the user may already have a remote instance of OpenShift 3 running with various applications on it and wants to monitor it. The IDE allows users to set up a connection to a remote instance of OpenShift 3 and then use logs (such as the application and build logs) to troubleshoot and monitor running applications. Use the following tasks to connect to and work with a remote OpenShift instance:
 
- - <<add_osv3,Add the OpenShift 3 Tooling to the IDE>>
- - <<set_binaries,Set up OpenShift Client Binaries>>
- - <<port_forwarding,Set up Port Forwarding>>
- - <<stream_app_logs,Stream Application Logs>>
- - <<stream_build_logs,Stream Build Logs>>
+. <<setup_binaries, Set up OpenShift Client Binaries>>
+. <<port_forwarding, Set up Port Forwarding>>
+. <<pod_logs, Stream Pod Logs>>
+. <<build_logs, Stream Build Logs>>
```

Comment:
I would change the order to 
- setup binaries
- build logs
- pod logs
- port forwarding